content/en/blog/_posts/2021-12-15-pod-security-admission-psa-beta.md

```diff
@@ -0,0 +1,620 @@
+
+In addition to modes you can also pin the policy to a specific version for example v1.22. Pinning to a specific version allows the behavior to remain consistent as policy changes happen over Kubernetes releases.
+
+## Hands on demo
+
+### Prerequisites
+
+- [kind CLI installed](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/getting-started/installation) container runtime & CLI
+
+### Deploy a kind cluster
+
+```shell
+kind create cluster --image kindest/node:v1.23.0
+```
+
+It might take awhile to start and once it's started it might take a minute or so before the node becomes ready.
+
+```shell
+kubectl cluster-info --context kind-kind
+```
+
+Wait for the node STATUS to become ready.
+
+```shell
+kubectl get nodes
+```
+
+The output is similar to this:
+
+```shell
+NAME                 STATUS   ROLES                  AGE   VERSION
+kind-control-plane   Ready    control-plane,master   54m   v1.23.0
+```
+
+### Confirm PSA is enabled
+
+The best way to [confirm the API's default enabled plugins](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#which-plugins-are-enabled-by-default) is to check the Kubernetes API container's help arguments.
+
+```shell
+kubectl -n kube-system exec kube-apiserver-kind-control-plane -it -- kube-apiserver -h | grep "default enabled ones"
+```
+
+The output is similar to this:
+
+```shell
+...
+      --enable-admission-plugins strings
+admission plugins that should be enabled in addition
+to default enabled ones (NamespaceLifecycle, LimitRanger,
+ServiceAccount, TaintNodesByCondition, PodSecurity, Priority,
+DefaultTolerationSeconds, DefaultStorageClass,
+StorageObjectInUseProtection, PersistentVolumeClaimResize,
+RuntimeClass, CertificateApproval, CertificateSigning,
+CertificateSubjectRestriction, DefaultIngressClass,
+MutatingAdmissionWebhook, ValidatingAdmissionWebhook,
+ResourceQuota).
+...
+```
+
+`PodSecurity` is listed in the group of default enabled admission plugins.
+
+### Configure Pod Security Admission
+
+Policies are applied to a namespace via labels. These labels are as follows:
+   * `pod-security.kubernetes.io/<MODE>: <LEVEL>` (required)
+   * `pod-security.kubernetes.io/<MODE>-version: <VERSION>` (*optional*, defaults to latest)
+
+A specific version can be supplied for each enforcement mode. The version pins the policy to the version that was defined at that Kubernetes version. This is to increase flexibility for future  Kubernetes enhancements and unknown security issues.
+
+From earlier, the modes are:
+   * `enforce` — Any Pods that violate the policy will be rejected
+   * `audit` — Pods with violations will be allowed and an audit annotation will be added
+   * `warn` — Pods that violate the policy will be allowed and a warning message will be sent back to the user.
```

Comment:
Rather than repeating this whole thing, maybe just enumerate the modes? 
```suggestion
From earlier, recall that the possible modes are `enforce`, `audit` and `warn`.
```